code/controllers/subsystem/minimaps.dm

```diff
@@ -320,6 +320,7 @@ SUBSYSTEM_DEF(minimaps)
 	var/minimap_displayed = FALSE
 	///Minimap object we'll be displaying
 	var/obj/screen/minimap/map
+	var/default_overwatch_level = 2
```

Comment:
uneeded

Reply:
actually, this could be usefull IF you're giving CIC a groundside minimap as well when overwatching.

Reply:
I figured this could be reused. I could try doing it for the CIC at a later date, but i prefer to have things a little split.

Reply:
I figured this could be reused. I could try doing it for the CIC at a later date, but i prefer to have things a little split.